.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 # Pull Request
 
 ## Checklist
-- [ ] Have you read {project_name}'s Code of Conduct? https://github.com/digicatapult/{project_module_name}/blob/main/CODE_OF_CONDUCT.md
+- [ ] Have you read Digital Catapult's [Code of Conduct](/../CODE_OF_CONDUCT.md)?
```

Comment:
Do we know if relative links like this will work as a .github?

Reply:
No idea, need to create a second repo to test it, and need this one to be in place to test it.

Reply:
If it doesn't work, then I can just open another PR which links to this repo's versions.